docs/sources/panels/transformations.md

```diff
@@ -259,6 +259,41 @@ Here is the result after applying the `Series to rows` transformation.
 | 2020-07-07 09:30:57 | Humidity    | 33    |
 | 2020-07-07 09:30:05 | Temperature | 19    |
 
+
+## Filter By Value
+
+This transformation allows you to filter your data directly in Grafana. This allows you to remove some of the data points from your query result. You can either include or exclude some of your data points (rows) from a field that match the filter test (`Equal`, `Greater`, `Lower`, `isNull`, `Regex`, `Equal` and a few more).
+
+Consider the following data set:
+
+| Time                | Temperature |
+|---------------------|--------------|
+| 2020-07-07 11:34:23 | 32    |
+| 2020-07-07 11:34:22 | 28    |
+| 2020-07-07 11:34:21 | 26   |
+| 2020-07-07 11:34:20 | 23    |
+| 2020-07-07 10:32:24 | 31    |
+| 2020-07-07 10:31:22 | 20    |
+| 2020-07-07 09:30:57 | 19    |
+
+For instance, you can `Include` all data points that have a **Temperature** lower than 30°C, you would get the following result :
```

Comment:
What would this look like in the UI or field? You have shown me what I can do, but not how to do it.

Also, why is Include in code format but Temperature is bold?

Reply:
Not in this case, since it is a table heading. The bold is intended to set UI text apart from other text in sentences.